website/docs/r/cdn_frontdoor_security_policy.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+subcategory: "CDN"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cdn_frontdoor_security_policy"
+description: |-
+  Manages a Frontdoor Security Policy.
+---
+
+# azurerm_cdn_frontdoor_security_policy
+
+Manages a Frontdoor Security Policy.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-cdn-frontdoor"
+  location = "West Europe"
+}
+
+resource "azurerm_cdn_frontdoor_profile" "example" {
+  name                = "example-profile"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_cdn_frontdoor_firewall_policy" "example" {
+  name                              = "exampleWAF"
+  resource_group_name               = azurerm_resource_group.example.name
+  cdn_frontdoor_profile_id          = azurerm_cdn_frontdoor_profile.example.id
```

Comment:
```suggestion
```

Reply:
Ahhh... good catch fixed... I thought I got all of those but apparently I missed one... Thanks 🚀

Reply:
not sure if you meant to take it out yet but it is still in there ;)

Reply:
@sebader lol... I was looking at the firewall policy documentation not the security policy documentation... I got it now... 🤣